frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -1,21 +1,90 @@
 import * as React from 'react';
-import { Alert, Title } from '@patternfly/react-core';
-import { K8sResourceKind, K8sKind } from '@console/internal/module/k8s';
+import * as _ from 'lodash';
+import { match } from 'react-router';
+import { Alert, Title, ActionGroup, Button } from '@patternfly/react-core';
+import { NodeKind, k8sPatch, k8sCreate, referenceForModel } from '@console/internal/module/k8s';
 import { ListPage } from '@console/internal/components/factory';
 import { NodeModel } from '@console/internal/models';
-import { ClusterServiceVersionKind } from '@console/operator-lifecycle-manager/src/types';
-import { NodeList } from './node-list';
-
+import { hasLabel, getName } from '@console/shared';
+import {
+  withHandlePromise,
+  HandlePromiseProps,
+  history,
+  FieldLevelHelp,
+  ButtonBar,
+} from '@console/internal/components/utils';
+import {
+  ocsRequestData,
+  OCS_LABEL,
+  labelTooltip,
+  minSelectedNode,
+} from '../../constants/ocs-install';
+import { OCSServiceModel } from '../../models';
+import { OCSStorageClassDropdown } from '../modals/storage-class-dropdown';
+import { OSDSizeDropdown } from '../../utils/osd-size-dropdown';
+import NodeTable from './node-list';
 import './ocs-install.scss';
 
-export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
-  namespace,
-  clusterServiceVersion,
-}) => {
-  const ListComponent = React.useCallback(
-    (nodeProps) => <NodeList {...nodeProps} ocsProps={{ namespace, clusterServiceVersion }} />,
-    [namespace, clusterServiceVersion],
+const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
+  const patch = [
+    {
+      op: 'add',
+      path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
```

Comment:
Let's not hardcode. There is a function which changes `/` to `~`. Let use the function and do string concatenation on OCS_LABEL.